### ISSUE TITLE
Compilation fails on main

### ISSUE THREAD

**Issue by benwilson512** [hissssst/tria#2]
This project looks exciting! I pulled down main and I'm getting the following issue:

Platform: OS X
Erlang: 25.1.1
Elixir: 1.14.1

```
==> tria
Compiling 35 files (.ex)

== Compilation error in file lib/tria/compiler/translator/elixir.ex ==
** (SyntaxError) lib/tria/compiler/translator/elixir.ex:225:35: syntax error before: '}'
    (elixir 1.12.3) lib/kernel/parallel_compiler.ex:319: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
```

**Comment by hissssst** [comment]
Yeah, it looks like 1.14 bug, I'll create an issue

**Comment by hissssst** [comment]
@benwilson512, apparently it was fixed in 1.14.2, so you can try this version
